Refresh.GameServer/Endpoints/Game/DataTypes/Response/GameLevelResponse.cs

```diff
@@ -44,9 +44,10 @@ public class GameLevelResponse : IDataConvertableFrom<GameLevelResponse, GameLev
     [XmlElement("playCount")] public required int TotalPlayCount { get; set; }
     [XmlElement("uniquePlayCount")] public required int UniquePlayCount { get; set; }
 
-    [XmlElement("yourDPadRating")] public int YourRating { get; set; }
+    [XmlElement("yourDPadRating")] public int YourDPadRating { get; set; }
```

Comment:
Why the change? I think `YourRating` was clear enough, especially given that LBP2 is our primary target

Reply:
With the suggestion of `YourStarRating`, i agree its unnessesary to change now, will revert